### DISCUSSION TITLE
提问答疑

### DISCUSSION THREAD

**Discussion by bunny-sadako** [datawhalechina/learn-python-the-smart-way-v2#20] [提问答疑]
w11  python3.9
<img width="431" alt="image" src="https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/128591902/81a30954-dc66-4b73-9eae-18019c67a61a">
<img width="350" alt="image" src="https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/128591902/b14564c5-0ef5-47a5-9c48-b97718a34638">
为什么if不对，换成while就好了？

**Comment by bunny-sadako** [comment]
那两个SS是后面误打上去的

**Comment by anine09** [comment]
你可能弄混了 for 和 if
